### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes several typos I spotted in the documentation.

### CODE REVIEW

File: .github/workflows/license_check.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     name: Scan code for licenses
     steps:
     - name: Checkout the code
-      uses: actions/checkout@v1
+      uses: actions/checkout@v3
```
Comment (kartben) [inline .github/workflows/license_check.yml:11]: Nice catch but this should be in a different commit please, not really a "typo".

File: doc/connectivity/bluetooth/api/cap.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .. _bluetooth_cap:
 
-Commmon Audio Profile
+Common Audio Profile
```
Comment (kartben) [inline doc/connectivity/bluetooth/api/cap.rst:3]: Remove now extra `#` from next line

**Review by kartben — CHANGES_REQUESTED** [review]
Thanks for this! Just a couple comments. Also please fix the compliance check by amending your commit message to follow the contributing guidelines.

File: doc/connectivity/bluetooth/api/cap.rst (hunk 2 above)
Comment (deining) [reply doc/connectivity/bluetooth/api/cap.rst:3]: Removed extra `#` as requested.

File: .github/workflows/license_check.yml (hunk 1 above)
Comment (deining) [reply .github/workflows/license_check.yml]: Reverted the change, will address this in a separate issue.

**Comment by deining** [comment]
> Thanks for this!

You are welcome.

> Just a couple comments.

These comments are all addressed now.

> Also please fix the compliance check by amending your commit message to follow the contributing guidelines.

Done.

**Review by kartben — APPROVED** [review]
Thanks!

**Review by jhedberg — APPROVED** [review]
